bluej/build.gradle

```diff
@@ -36,39 +36,70 @@ dependencies {
     testImplementation 'org.hamcrest:hamcrest-library:1.3'
     testImplementation 'org.testfx:testfx-core:4.0.15-alpha'
     testImplementation 'org.testfx:testfx-junit:4.0.15-alpha'
-    testImplementation 'org.testfx:openjfx-monocle:jdk-12.0.1+2'
+    testImplementation 'io.github.sebivenlo:openjfx-monocle:jdk-21.0.1'
+    testImplementation 'junit:junit:4.13.2'
 }
 
 compileJava {
     options.compilerArgs += [
         "-Xplugin:threadchecker.TCPlugin",
         // we use some private packages/classes currently:
-        "--add-exports=javafx.graphics/com.sun.glass.ui=ALL-UNNAMED"]
+        // TODO: Without commenting this out I get the following error when trying to run tests:
+        //       error: exporting a package from system module javafx.graphics is not allowed with --release
+        // "--add-exports=javafx.graphics/com.sun.glass.ui=ALL-UNNAMED"
```

Comment:
If I leave that in, then I can't run tests due to `--release` disallowing use of those parameters. I'm not sure if that's something that should happen or not, but it precludes running tests on my machine.